### PR TITLE
Lisa workspace

### LINKED ISSUE #1: Footer SVG
Add SVG logo to footer

### LINKED ISSUE #2: Footer icons
Add social media icons to footer

### LINKED ISSUE #3: Footer text
Add text links to footer

### LINKED ISSUE #4: Footer div
Dimensions for footer

### LINKED ISSUE #15: Add type SVG files

### PR DESCRIPTION
**Pull request by ljwedin**
Fixed merge conflicts in VSC, deleted remote branch and reuploaded this one. Added SVG files for text as well.

### CODE REVIEW

File: style.scss [hunk 1]
```diff
@@ -121,4 +131,57 @@ $medium: 768px;
...
+    flex-direction: column-reverse;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+}
+
+.footerNav {
+    list-style: none;
+    margin-bottom: 35px;
+    & li {
```
Comment (ssmeds) [inline style.scss:151]: Is it necessary to use the ampersand inside of the .footerNav class? I think it's okey to just write li.

**Review by ssmeds — COMMENTED** [review]
See my comment about the ampersand.

File: style.scss (hunk 1 above)
Comment (ljwedin) [reply style.scss:151]: You're right, my mistake!

Comment (ljwedin) [reply style.scss:151]: You are right! I didn't know you could omit the ampersand. I'll merge for now but keep that in mind in the future.

**Comment by LinusZachrisson** [comment]
looks good!

**Pull request merged**